minimesos/src/main/java/com/containersol/minimesos/cluster/MesosMaster.java

```diff
@@ -1,5 +1,6 @@
 package com.containersol.minimesos.cluster;
 
+import com.containersol.minimesos.state.State;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.containersol.minimesos.state.State'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)